src/main/kotlin/org/rust/cargo/toolchain/RustToolchain.kt

```diff
@@ -108,31 +100,16 @@ data class RustToolchain(val location: String) {
     }
 }
 
+open class Version(val major: Int, val minor: Int, val patch: Int) {
+    override fun toString(): String = "$major.$minor.$patch"
+}
```

Comment:
looks like a duplicate of [SemVer](https://upsource.jetbrains.com/idea-ce/file/idea-ce-7771e0e8bc3c66a436063345d56b46f2fc670cd4/platform/util/src/com/intellij/util/text/SemVer.java?nav=0:0:focused) class